crates/db_schema/src/impls/post.rs

```diff
@@ -169,6 +180,7 @@ impl Post {
     let object_id: DbUrl = object_id.into();
     post::table
       .filter(post::ap_id.eq(object_id))
+      .filter(post::scheduled_time.is_null())
```

Comment:
Not sure if this one makes sense. You should be able to update an unpublished post (say you want to change the title, or even the publish time). The API will have to do the validity checks.

Reply:
Your comment is on read_from_apub_id(). This ensures that scheduled posts cannot be federated prematurely.